packages/app/source/server/projections/open-appeals-projection.js

```diff
@@ -0,0 +1,36 @@
+Space.eventSourcing.Projection.extend(Donations, 'OpenAppealsProjection', {
```

Comment:
Then let's go one step further and consolidate publications into a single file, but I have to disagree that combining the public Appeals view cache with an administrators view cache is premature optimisation. I can understand if the exact same events are used to populate two collections, but here we're talking two very different views, almost in different applications conceptually. One only cares about open appeals then (will) remove once closed or fulfilled, and the other cares about the full lifecycle. I'd have no problem consolidating the collections into a single file, as this is the _real_ boilerplate IMO :wink: